### PR TITLE
docs: sd-display docs [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Comment by coraliefeil** [comment]
I’ve refined [Figma](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2705-24820&node-type=SECTION&t=SwfvfYHM9JnxEX6r-11).

@MartaPintoTeixeira based on figma - could you review code, pls? THX

File: packages/components/src/styles/display/display.stories.ts [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
 const { generateTemplate } = storybookTemplate('sd-display');
 
 /**
- * Display provides larger text sizes that are not used as headlines. <br>
- * <br>
- * The different sizes allow for a more versatile styling of text elements. Display text should not be used as substitute for headlines.<br>
- * <br>
- * <b>Sizes</b>
- * <li>xl is the default size.</li>
+ * Provides larger text sizes that are not used as headlines.
```
Comment (yoezlem) [inline packages/components/src/styles/display/display.stories.ts:10]:
```suggestion
 * Used to provide larger text sizes that are not used as headlines.
```

File: packages/components/src/styles/display/display.stories.ts [hunk 2]
```diff
@@ -43,21 +35,28 @@ export const Default = {
...
+ * - `sd-display--size-3xl`
+ * - `sd-display--size-4xl`
+ */
+
+export const Size = {
+  render: () => html`
+    <div class="flex flex-col gap-6">
+      <div class="sd-display sd-display--size-default">Default</div>
+      <div class="sd-display sd-display--size-3xl">3xl</div>
+      <div class="sd-display sd-display--size-4xl">4xl</div>
```
Comment (yoezlem) [inline packages/components/src/styles/display/display.stories.ts:49]:
```suggestion
      <div class="sd-display sd-display--size-default">Lorem ipsum</div>
      <div class="sd-display sd-display--size-3xl">Lorem ipsum</div>
      <div class="sd-display sd-display--size-4xl">Lorem ipsum</div>
```

File: packages/components/src/styles/display/display.stories.ts [hunk 3]
```diff
@@ -43,21 +35,28 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a display with inverted colors.
+ * Use the `sd-display--size-` class to set the display’s size:
```
Comment (yoezlem) [inline packages/components/src/styles/display/display.stories.ts:38]: Please align with Design.
Text here is: Use “sd-display” modifiers for alternative appearances.

File: packages/components/src/styles/display/display.stories.ts [hunk 4]
```diff
@@ -43,21 +35,28 @@ export const Default = {
...
-        y: [{ type: 'attribute', name: 'sd-display--inverted', values: [false, true] }]
-      },
-      constants: { type: 'attribute', name: 'sd-display--inverted', value: true },
-      options: {
-        templateBackgrounds: { alternate: 'y', colors: ['transparent', 'rgb(var(--sd-color-primary, 0 53 142))'] }
-      },
-      args
-    });
+  render: () => {
+    return html`<div class="p-4 bg-primary"><div class="sd-display sd-display--inverted">Inverted</div></div>`;
```
Comment (yoezlem) [inline packages/components/src/styles/display/display.stories.ts:60]: Please add all three sizes:

image

Comment (auroraVasconcelos) [reply packages/components/src/styles/display/display.stories.ts]: Fixed according to updated version from figma:
image

**Review by smfonseca — APPROVED** [review]

File: packages/components/src/styles/display/display.stories.ts [hunk 5]
```diff
@@ -43,21 +35,28 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a display with inverted colors.
+ * Use `sd-display` modifiers for alternative appearances.
+ * - `sd-display--size-default`
+ * - `sd-display--size-3xl`
+ * - `sd-display--size-4xl`
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/display/display.stories.ts:41]:
```suggestion
 * - `sd-display--size-4xl`: 4xl can be used as an alternative
```

File: packages/components/src/styles/display/display.stories.ts [hunk 6]
```diff
@@ -43,21 +35,28 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a display with inverted colors.
+ * Use `sd-display` modifiers for alternative appearances.
+ * - `sd-display--size-default`
+ * - `sd-display--size-3xl`
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/display/display.stories.ts:40]:
```suggestion
 * - `sd-display--size-3xl`: 3xl can be used as an alternative
```

File: packages/components/src/styles/display/display.stories.ts [hunk 7]
```diff
@@ -43,21 +35,28 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a display with inverted colors.
+ * Use `sd-display` modifiers for alternative appearances.
+ * - `sd-display--size-default`
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/display/display.stories.ts:39]:
```suggestion
 * - `sd-display--size-default`:  is the default display size
```

File: packages/components/src/styles/display/display.stories.ts [hunk 8]
```diff
@@ -43,21 +35,28 @@ export const Default = {
...
+    <div class="flex flex-col gap-6">
+      <div class="sd-display sd-display--size-4xl">Lorem ipsum</div>
+      <div class="sd-display sd-display--size-3xl">Lorem ipsum</div>
+      <div class="sd-display sd-display--size-default">Lorem ipsum</div>
+    </div>
+  `
+};
+
+/**
+ * Use the `sd-display--inverted` class to make a display with inverted colors.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/display/display.stories.ts:55]:
```suggestion
 * Use the `sd-display--inverted` class when displayed on primary background.
```

**Review by Vahid1919 — CHANGES_REQUESTED** [review]

File: packages/components/src/styles/display/display.stories.ts [hunk 9]
```diff
@@ -43,21 +38,29 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a display with inverted colors.
+ * Use `sd-display` modifiers for alternative appearances.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/display/display.stories.ts]:
```suggestion
 * Use `&--size` class for alternative appearances.
```

File: packages/components/src/styles/display/display.stories.ts [hunk 10]
```diff
@@ -1,22 +1,21 @@
...
- * Display provides larger text sizes that are not used as headlines. <br>
- * <br>
- * The different sizes allow for a more versatile styling of text elements. Display text should not be used as substitute for headlines.<br>
- * <br>
- * <b>Sizes</b>
- * <li>xl is the default size.</li>
+ * Used to provide larger text sizes that are not used as headlines.
+ *
+ * **Related templates**:
+ * - [Headline, Display and Leadtext with Mark](?path=/docs/templates-headline-display-and-leadtext-with-mark--docs)
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/display/display.stories.ts:13]: Template name should be: "Headline with Mark"

Comment (auroraVasconcelos) [reply packages/components/src/styles/display/display.stories.ts]: Updated

**Review by smfonseca — APPROVED** [review]

**Review by MartaPintoTeixeira — APPROVED** [review]

**Comment by coraliefeil** [comment]
Approved <3

**Pull request merged**